### DISCUSSION TITLE
How is it possible to have the slurm launcher run a batch in detached mode?

### DISCUSSION THREAD

**Discussion by akazemipour** [facebookresearch/hydra#2702] [Q&A] [answered]
Hi everyone!
Thank you for your amazing project!
I wanted to know if it's possible to have the functionality of `slurm.sbatch` in submitit launcher so, that after running the program it's run in the background and it's `detached` from the terminal, let's say.

**Comment by odelalleau** [comment, marked as answer]
NB: this is discussed in issue #2703
